### PR TITLE
test: Solve flag and segments flaky tests

### PR DESCRIPTION
**Pull request by novakzaballa**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have added information to `docs/` if required so people know about the feature!
- [ ] I have filled in the "Changes" section below?
- [ ] I have filled in the "How did you test this code" section below?
- [ ] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

_Please describe._

## How did you test this code?

_Please describe._

### CODE REVIEW

**Comment by kyle-ssg** [comment]
image

Does this suggest the flakiness has not been solved?

File: frontend/e2e/helpers.cafe.ts [hunk 1]
```diff
@@ -295,6 +295,10 @@ export const createFeature = async (
   description = 'description',
 ) => {
   await gotoFeatures()
+  await t.eval(() => {
```
Comment (kyle-ssg) [inline frontend/e2e/helpers.cafe.ts:298]: Hmm, what's the theory behind this solving issues with e2e ?

Comment (novakzaballa) [reply frontend/e2e/helpers.cafe.ts:298]: That function will scroll down the screen. One issue is that I was trying to click on a component that wasn't visible.

**Comment by novakzaballa** [comment]
> image
> 
> Does this suggest the flakiness has not been solved?

@kyle-ssg This is another flaky issue. The problem here is that, for some reason, the ID 'email' was not found.

**Review by kyle-ssg — APPROVED** [review]

**Pull request closed without merging**